src/api.js

```diff
@@ -163,4 +175,15 @@ module.exports = {
 			self.CHOICES_INPUTS.push({ id: input.uri, label: input.title })
 		}
 	},
+
+	buildAppList: function () {
+		let self = this
+
+		self.CHOICES_APPS = []
+
+		for (let i = 0; i < self.DATA.apps.length; i++) {
+			let app = self.DATA.apps[i]
+			self.CHOICES_APPS.push({ id: app.uri, label: entities.decodeHTML5(app.title) })
```

Comment:
It seems the version specific one is a legacy alias:
```suggestion
			self.CHOICES_APPS.push({ id: app.uri, label: entities.decodeHTML(app.title) })
```

See:
https://github.com/fb55/entities/blob/f1fd10a5f63925504c6057223d6054697243407a/src/index.ts#L162-L164